examples/cluster/main.tf

```diff
@@ -0,0 +1,49 @@
+provider "vmc" {
+  refresh_token = var.api_token
+  org_id = var.org_id
+}
+
+# Empty data source defined in order to store the org display name and name in terraform state
+data "vmc_org" "my_org" {
+}
+
+data "vmc_connected_accounts" "my_accounts" {
+  account_number = var.aws_account_number
+}
+
+data "vmc_customer_subnets" "my_subnets" {
+  connected_account_id = data.vmc_connected_accounts.my_accounts.id
+  region               = replace(upper(var.sddc_region), "-", "_")
+}
+
+resource "vmc_sddc" "sddc_1" {
+  sddc_name           = var.sddc_name
+  vpc_cidr            = var.vpc_cidr
+  num_host            = var.num_hosts
```

Comment:
maybe we should rename num_hosts in variables.tf to sddc_num_hosts to avoid confusion with cluster num hosts

Reply:
Done